### PR TITLE
[1/x] Change venft form to use an int value for ease of parsing

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Change staking form from using FormattedNumberInput with a string datatype to use an antd InputNumber instead. There were numerous problems being caused by decorators in the string value causing issues with parseWad.

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by jmilldotdev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1586** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1586" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1586?utm_source=stack-comment).

**Review by tomquirk — APPROVED** [review]

**Comment by jmilldotdev** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1586?utm_source=stack-comment).


Job Reference: 5peKCVtRG7hIeC6BSrBI

**Pull request merged**
